ansible/roles/cni/tasks/main.yml

```diff
@@ -1,11 +0,0 @@
----
-- name: make cni directory
-  file:
-    dest: /opt/cni
-    state: directory
-
-- name: download and unpack the cni plugins
-  unarchive:
-    src: "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-amd64-{{ cni.release }}.tar.gz"
-    dest: /opt/cni
-    remote_src: yes
```

Comment:
This role appears vestigial, since we are installing kubernetes-cni packages in the kubernetes role.

Reply:
Good catch.